pkg/controllers/cassandra/nodepool/nodepool.go

```diff
@@ -68,6 +79,61 @@ func (e *defaultCassandraClusterNodepoolControl) removeUnusedStatefulSets(
 	return nil
 }
 
+// Return the lowest int from a slice of ints
+func min(v []int) (m int) {
+	if len(v) > 0 {
+		m = v[0]
+	} else {
+		panic("min called with empty slice")
+	}
+	for _, e := range v {
+		if e < m {
+			m = e
+		}
+	}
+	return
+}
+
+func (e *defaultCassandraClusterNodepoolControl) labelSeedNodes(
+	cluster *v1alpha1.CassandraCluster,
+	set *appsv1beta1.StatefulSet,
+) error {
+	var nodepoolPods []*v1.Pod
+	allPods, err := e.pods.Pods(cluster.Namespace).List(labels.Everything())
+	if err != nil {
+		return err
+	}
+	for _, pod := range allPods {
+		if metav1.IsControlledBy(pod, set) {
+			nodepoolPods = append(nodepoolPods, pod)
+		}
+	}
+
+	// if there are no pods, return early
+	if len(nodepoolPods) < 1 {
+		return nil
+	}
+
+	// Choose the lowest StatefulSet member and mark it as a seed
+	numberedPods := make(map[int]*v1.Pod)
+	ns := []int{}
+	for _, p := range nodepoolPods {
+		elements := strings.Split(p.Name, "-")
+		n, err := strconv.Atoi(elements[len(elements)-1])
+		if err != nil {
+			return err
+		}
+		numberedPods[n] = p
+		ns = append(ns, n)
+	}
+
+	pod := numberedPods[min(ns)]
```

Comment:
:grey_question: There are some useful utilities (`getOrdinal` and `[]*v1.Pod` sorted by ordinal) in https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/statefulset/stateful_set_utils.go but they're private :cry: 

I also found this issue (which may have been addressed) for exposing the pod ordinal as a  pod annotation.
 * https://github.com/kubernetes/kubernetes/pull/30860

Reply:
There's some parts of our codebase already (it may be as part of e2e test utils, or validation functions) for calculating the ordinal of a pod in a statefulset.

Reply:
... if it is in e2e test utils, that'll be as part of the ginkgo PR which hasn't merged yet come to think #242